### ISSUE TITLE
Update Nested List for Accessibility on resources/american-security-drone-act/

### ISSUE THREAD

**Issue by CodyHinze** [GSA/smartpay-website#579]
Please correct the accessibility testing failure on the nested list's resources/american-security-drone-act/ page.  

Per AMP and the WCAG, an order or unordered list should only contain LI elements.  This page currently contains an unordered list with an order list directly nested.  To correct this problem, the ordered list should be combined into the bullet above the nested list.

Compliant Example:
![Image](https://github.com/GSA/smartpay-website/assets/2073449/24ab09f6-6791-4451-bec3-fab7b56b8c8a)

Sample with Change:
![Image](https://github.com/GSA/smartpay-website/assets/2073449/89cd1472-d8b4-45b7-b613-85660d2997e4)

**Comment by JennaySDavis** [comment]
**#579 Acceptance Criteria**
 

| Pass/Fail | Description |
| --- | --- | 
|Pass |Nested List for Accessibility on Resources > American Security Drone Act Updated |


**Comments/Additional Notes**
N/A

**ADA Compliance (Automated scan via Chrome Lighthouse)**

| Criteria | Score |
| --- | --- | 
| Performance | 100 |
| Accessibility | 100 |
| Best Practices | 100 |



Passed 06/24/2024 - JSD

**Comment by johnbeallgsa** [comment]
Thanks for explaining this in the Demo. Moving to Done.
